python/py-virtualenv/Portfile

```diff
@@ -5,7 +5,7 @@ PortGroup           python 1.0
 PortGroup           select 1.0
 
 name                py-virtualenv
-version             16.7.9
+version             20.0.2
```

Comment:
since your PR the port has been updated to 20.0.3, perhaps do this update within this PR as well?